composer.json

```
@@ -16,8 +16,8 @@
         }
     ],
     "require": {
-        "php": "^7.1",
-        "illuminate/support": "^5.5|^6"
+        "php": "^7.2.5",
```

I'd recommend leaving support for PHP 7.1 since that's the supported version with laravel `^5.5`

```suggestion
        "php": "^7.1",
```